plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -353,8 +353,8 @@ def main():
 
         user_group_id = group.id
     else:
-        user = gitlab_instance.users.list(username=gitlab_instance.user.username)[0]
-        user_group_id = user.id
+        namespace = gitlab_instance.namespaces.list(search=gitlab_instance.user.username)[0]
+        user_group_id = namespace.id
```

Comment:
```suggestion
        namespace_group_id = namespace.id
```
(Same in other places where this is used.)